### PR TITLE
ESLint rule: disallow multiple parameters to classList.add

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Adds a custom ESLint rule disallowing from passing multiple parameters to `classList.add`. This is because IE10 and IE11 do not support multiple class names being passed, even with polyfills provided by polyfill.io (see #16673)

The ideal solution would actually be to not use this Lint rule and instead help to fix the real issue (Financial-Times/polyfill-service#1167)

## What is the value of this and can you measure success?

Reminds developer to implement manual workaround (multiple calls to `classList.add` 😭 )

## Does this affect other platforms - Amp, Apps, etc?

No

## Tested in CODE?

No

### CODE REVIEW

File: dev/eslint-plugin-guardian-frontend/rules/no-multiple-classlist-parameters.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                    ] = context.getSourceCode().getTokensAfter(node, 3);
+
+                    if (
+                        isDot(dot1) &&
+                        isIdentifierOf('classList', parent) &&
+                        isComma(nextToken)
+                    ) {
+                        context.report({
+                            node,
+                            message: 'You are using a classList.add',
```
Comment (sndrs) [inline dev/eslint-plugin-guardian-frontend/rules/no-multiple-classlist-parameters.js]: should this message say something about why we need only one arg? maybe link it to the polyfill.io issue?

**Comment by SiAdcock** [comment]
@sndrs Do you fancy giving this a quick check? Would be good to catch this issue in the short term until a fix for IE10/11 is developed

**Comment by sndrs** [comment]
sorry missed this! 👍

**Review by sndrs — APPROVED** [review]

**Comment by SiAdcock** [comment]
Thanks @sndrs!

**Pull request merged**